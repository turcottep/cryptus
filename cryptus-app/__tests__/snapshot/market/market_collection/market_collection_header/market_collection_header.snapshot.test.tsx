import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import MarketCollectionHeader from "../../../../../components/market/market_collection/market_collection_header/market_collection_header";

describe("<MarketCollectionHeader />", () => {
  it("should display properly", async () => {
    const tree = renderer.create(<MarketCollectionHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
