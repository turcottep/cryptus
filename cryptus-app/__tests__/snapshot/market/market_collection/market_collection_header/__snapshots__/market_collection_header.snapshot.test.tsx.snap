// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarketCollectionHeader /> should display properly 1`] = `
<div>
  <div>
    <a
      href="login"
      target="_blank"
    >
      <img
        src="/icons/favorite_icon.png"
      />
    </a>
    <a
      href="login"
      target="_blank"
    >
      <img
        src="/icons/notification_icon.png"
      />
    </a>
  </div>
  <div>
    <a
      onClick={[Function]}
      target="_blank"
    >
      <img
        src="/icons/x_icon.png"
      />
    </a>
  </div>
</div>
`;
